src/android/emulator/mod.rs

```diff
@@ -27,10 +27,10 @@ impl Emulator {
         &self.name
     }
 
-    pub fn start(&self, env: &Env) -> bossy::Result<bossy::Handle> {
```

Comment:
this is actually a breaking change, so we either need to add a new method or make this a minor bump.